apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3088,6 +3088,8 @@
   "List_of_Direct_Messages": "List of Direct Messages",
   "List_view": "List View",
   "Livechat": "Livechat",
+  "Livechat_AdditionalWidgetScripts": "Livechat Widget Additional Scripts. Its possible to specify a CSV list of scripts, for example : https://yourUrl/customScrip1.js,https://yourUrl/customScrip2.js",
+  "Livechat_WidgetLayoutClasses": "Additional classes to add in the body section of the livechat widget. Its possible to specify a CSV list of classes, for example : https://yourUrl/customFile1.css,https://yourUrl/customFile2.css",
```

Comment:
Keep the name of the settings short and concise. Use the `_Description` of the setting to put examples and more data.